src/content/docs/en/guides/view-transitions.mdx

```diff
@@ -217,3 +217,9 @@ import { ViewTransitions } from 'astro:transitions';
 :::note[known limitations]
 The `morph` animation cannot be simulated in traditional CSS. So any element using this animation will not be animated.
 :::
+
+## `prefers-reduced-motion`
+
+You may be concerned about how all these animations will affect users that have [`prefer-reduced-motion`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion) enabled on their device. Don't worry! Currently, Astro includes a CSS media query that disables *all* view transition animations whenever this setting is detected. Instead, the browser will simply swap the DOM elements without an animation.
+
+If you would like to control your animations more granularly in these situations, you will need to use the API by itself and make sure that the `<ViewTransitions />` component is not included anywhere on the page. While this may be a bit of an over-correction, we want to ensure that we accommodate these users as they may have this setting enabled due to health concerns. And, we're open to suggestions if you have a better idea for covering these accessibility issues. As always, pull requests are welcome!
```

Comment:
```suggestion
Astro's `<ViewTransitions />` component includes a CSS media query that disables *all* view transition animations whenever the [`prefer-reduced-motion`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion) setting is detected. Instead, the browser will simply swap the DOM elements without an animation.
```
Especially on experimental pages where the features and how they work are not only very new but also subject to change, less is more! We just want to provide a matter-of-fact "here's what is true" with the information super easy to find, not lost in longer text.

I like your instinct to suggest a workaround, but then it's not really an "Astro view transitions" thing anymore, and we're not responsible for documenting how *not* to use view transitions here!  :smile: